medusa/notifiers/emailnotify.py

```diff
@@ -65,15 +65,15 @@ def test_notify(self, host, port, smtp_from, use_tls, user, pwd, to):
         msg['Date'] = formatdate(localtime=True)
         return self._sendmail(host, port, smtp_from, use_tls, user, pwd, [to], msg, True)
 
-    def notify_snatch(self, title, message):
+    def notify_snatch(self, title, message, ep_obj):
```

Comment:
I think you forgot to make it a keyword argument.

Reply:
I kinda did this on purpose. I had it as keyword first. But it's always called with an ep_obj. So no need to make it default to None.

Reply:
But doesn't this defeat the purpose to use **kwargs for all other notifiers? We are forced to always add the `email` argument for all notifiers like this.

Reply:
No, the **kwargs param just prevents them from Errorring. As it's now its not causing any errors.

Reply:
The point is that it is pretty confusing right now. The method header should be equal for all and the arguments should be handled inside the function IMO.